shelley/chain-and-ledger/executable-spec/src/STS/Ledger.hs

```diff
@@ -89,3 +95,87 @@ instance
   => Embed (UTXOW crypto) (LEDGER crypto)
  where
   wrapFailed = UtxowFailure
+
+-- |In the case where a transaction is valid for a given ledger state,
+-- apply the transaction as a state transition function on the ledger state.
+-- Otherwise, return a list of validation errors.
+asStateTransition
```

Comment:
the goal is to eventually delete these too, right? it's fine if it happens in another PR, etc, just making sure I follow. :)

Reply:
yes, this was to first make sure that the existing tests still run, but the functionality used is the STS.
This actually required some changes, e.g., non-zero maximal transaction sizes which wasn't done in the original tests / examples.

Reply:
but now I think it would be better to at least move the function to the test package